client/src/Validation/validation.js

```
@@ -483,8 +483,71 @@
 	}),
+
+	// ntfy-specific fields
+	ntfyAuthMethod: joi.when("type", {
+		is: "ntfy",
+		then: joi.string().valid("none", "username_password", "bearer_token").default("none"),
+		otherwise: joi.forbidden(),
+	}),
+
+	ntfyUsername: joi.when("type", {
+		is: "ntfy",
+		then: joi.when("ntfyAuthMethod", {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)